Interpreter.Eval fails if the last line of the input is a comment

If `Interpreter.Eval` is called with `s` and `s` is not terminated with a newline and the last line of `s` contains `//`, evaluation fails because the closing brace of the generated main function is commented out.

```go
package main

import "github.com/traefik/yaegi/interp"

func main() {
	i := interp.New(interp.Options{})
	_, err := i.Eval(`
		println("Hello world")
		// foo bar`)
	if err != nil {
		panic(err)
	}
}
```

Expected result:
```console
$ go run ./sample.go
Hello world
```

Got:
```console
$ go run ./sample.go
panic: _.go:3:14: expected '}', found 'EOF'

goroutine 1 [running]:
main.main()
        .../sample.go:11 +0xbb
exit status 2
```

--- a/interp/ast.go
+++ b/interp/ast.go
@@ -357,7 +357,7 @@ func ignoreError(err error, src string) bool {
 }
 
 func wrapInMain(src string) string {
-	return fmt.Sprintf("package main; func main() {%s}", src)
+	return fmt.Sprintf("package main; func main() {%s\n}", src)
 }
 
 // Note: no type analysis is performed at this stage, it is done in pre-order

--- a/interp/interp_eval_test.go
+++ b/interp/interp_eval_test.go
@@ -1504,3 +1504,10 @@ func TestStdio(t *testing.T) {
 		t.Fatalf("%v not *os.file", v.Interface())
 	}
 }
+
+func TestIssue1142(t *testing.T) {
+	i := interp.New(interp.Options{})
+	runTests(t, i, []testCase{
+		{src: "a := 1; // foo bar", res: "1"},
+	})
+}

